File: array and string/reverse_words_in_a_string_III.py
class Solution:
    def reverseWords(self, s: str) -> str:
        l = list(s.split())
        l2 = []
        for i in l:
            l2.append(i[::-1])
        s1 = " ".join(i for i in l2)
        return s1
        

File: array and string/reverse_words_in_a_string_III.java
class Solution {
    public String reverseWords(String s) {
        String[] arr = s.trim().split(" "); // Replace multiple white space with single white space using regex
    	
    	String ans = "";
    	
    	for(int i=0; i<arr.length; i++)
    	{
    		StringBuilder input1 = new StringBuilder(); 

            // append a string into StringBuilder input1 
            input1.append(arr[i]); 

            // reverse StringBuilder input1 
            input1 = input1.reverse(); 
            
            ans = ans + input1+ " ";
    	}
    	
    	return ans.trim();
    	
    }
}